### PR TITLE
add documentation for jolokia()

### PR DESCRIPTION
**Pull request by vetinari**
also: fix inconsistent headings warning in scalyr

### CODE REVIEW

**Comment by vetinari** [comment]
:+1:

**Comment by vetinari** [comment]
...fixes #10

File: docs/user/check-ref/http_wrapper.rst [hunk 1]
```diff
@@ -96,3 +96,25 @@ The object returned by the :py:func:`http` function provides methods: :py:meth:`
     Parse the resulting text result according to the Prometheus specs using their prometheus_client.
 
         http("/metrics", timeout=5).prometheus()
+
+.. _http-jolokaia:
```
Comment (mohabusama) [inline docs/user/check-ref/http_wrapper.rst]: Typo: ``jolokia``

**Comment by vetinari** [comment]
:+1:

**Comment by mohabusama** [comment]
👍

**Pull request merged**